src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -675,8 +675,18 @@ def get_sp_auth_info(self, subscription_id=None, name=None, password=None, cert_
     def get_installation_id(self):
         installation_id = self._storage.get(_INSTALLATION_ID)
         if not installation_id:
-            import uuid
-            installation_id = str(uuid.uuid1())
+            try:
+                # We share the same installationId with Azure Powershell. So try to load installationId from PSH file
+                # Contact: DEV@Nanxiang Liu, PM@Damien Caro
+                shared_installation_id_file = os.path.join(self.cli_ctx.config.config_dir, 'AzureRmSurvey.json')
```

Comment:
```suggestion
                shared_installation_id_file = os.path.join(self.cli_ctx.config.config_dir, 'AzureRmContextSettings.json')
```
PSH has changed the file name. @Nickcandy for confirmation